src/app/modules/evaluate/component/evaluate-options/evaluate-options.component.ts

```diff
@@ -50,12 +50,16 @@ export class EvaluateOptionsComponent implements OnInit {
   }
 
   public onToggleOnlineClick(): void {
-    this.options.online = !this.options.online;
     this.optionsChange.emit(this.options);
   }
 
   public onLeaguesWheel(event: WheelEvent, leagues: LeagueMap): void {
     const factor = event.deltaY > 0 ? -1 : 1;
+
+    this.changeLeague(factor, leagues);
+  }
+
+  public changeLeague(factor, leagues) {
```

Comment:
👍 to the refactoring here, just wanted to note that some of these added methods have types and some do not - I think if we're going to use types / TypeScript, we should be consistent (here and `getIndexText` below).

Reply:
I think this should be a failure in tslint.

https://github.com/PoE-Overlay-Community/PoE-Overlay-Community-Fork/blob/master/tslint.json#L57

Reply:
Maybe we could make `npm run ng:lint` or `npm run format` a pre-push hook in git if this would be caught there.

Reply:
kifers, i agree. i need to add those types, definitely was being lazy on that one. I am used to a pipeline and intellisense. I will work on the eslint in IDE issue, and try to do better at manually running my linter. xD

Reply:
linting applied